Resources/Prototypes/_Stalker_EN/ShopPresets/NPCShops/Barman.yml

```diff
@@ -202,17 +202,38 @@
         STEnergyDrink: 100
         STEnergyDrinkMonster: 200
         STEnergyDrinkRevo: 150
-        STDrinkBeerCanBaltic: 50
-        DrinkVodkaBottleFull: 75
         STFoodBreadPlain: 20
         DrinkFlaskOld: 30
         DrinkWaterBottleFull: 50
         FoodTinMRE: 100
         DrinkLemonadehBottleFull: 75
         STFoodMeatSalami: 125
 
-    - name: shop-category-armor
+    - name: shop-category-provisions # If you are buying these, you don't need the money
```

Comment:
People could/would be using this for roleplay purposes, why does it have to be 500 rubles for a GLASS, not a bottle of one? Also 6969 haha, funny number.